CHANGELOG.md

```diff
@@ -1,3 +1,150 @@
+## 1.14.0 (2019-08-21)
+
+This feature release adds a read-only mode to the Flux daemon, adds
+support for mapping images in `HelmRelease` resources using YAML dot
+notation annotations, eases the deployment of Flux with a new `fluxctl
+install` command which generates the required YAML manifests, lots of
+documentation improvements, and many more.
+
+### Fixes
+
+- Fetch before branch check to detect upstream changes made after the
+  initial clone
+  [fluxcd/flux#2371][]
+
+### Enhancements
+
+- With `--git-readonly`, `fluxd` can now sync a git repo without having
+  write access to it. In this mode, `fluxd` will not make any commits
+  to the repo.
+  [fluxcd/flux#1807][]
+- Mapping images in `HelmRelease resources` using YAML dot notation
```

Comment:
Say "dot notation annotation" ten times, fast :-)

Reply:
It is my new favourite word combination.

Reply:
Then it is decided: henceforth we shall exploit all opportunities to use the term "dot notation annotation"